src/components/Observations/ObservationsDataView.tsx

```diff
@@ -13,44 +16,80 @@ import ObservationMapView from './map/ObservationMapView';
 export type ObservationsDataViewProps = SearchProps & {
   setFilterOptions: (value: FieldOptionsMap) => void;
   selectedPlantingSiteId: number;
```

Comment:
If we are now passing the selectedPlantingSite, we can get rid of this prop and get it from selectedPlantingSite

Reply:
The selected planting site may be undefined. The site id also captures the -1 “All” value

Reply:
I realized that after writing the comment haha